### ISSUE TITLE
[BUG]: Time warp to specific time (maneuver, SOI, Ap/Pe) sometimes does not speed up

### ISSUE THREAD

**Issue by Tetopia** [Simpit-team/KerbalSimpitRevamped#86]
# Description of the bug

Mod version: Simpit V2.2.0
Game version: 1.12.4 / 1.12.5
Operating system: Windows 10
Microprocessor/s in use: Arduino Pro Micro Clone

### What feature are you trying to use that is in the documentation, that is not behaving as the documentation says it should/will do?  
When I press my button to try and warp to the next maneuver it displays the "TimeWarping... T- xxminutes. Press [Minus] to cancel" message in game (with xx beeing the correct number), but at the same time it sets the actual timewarp to x1 (normal time, without timewarp). 
<br>

### What have you done to ensure the issue is a bug?
The mod/game obviously recognizes that I want to timewarp to a specific time, but the warp rate is wrong.
<br>

### How repeatable is it? 
Sometimes it does behave how it should and speeds up the time. I have not found a pattern yet how to reliably reproduce the issue. It might have to do with how far the point in time to warp to is in the future. Short distances (like about 5 minutes) seem to make more problems than medium distances (like about 20 minutes).
<br>

## To Reproduce
Use the following code on an arduino and connect a button to pin 2. Start KSP and go to a ship e.g. in orbit. Generate a maneuver 3 minutes in the future and try warping to it by pressing the button connected to pin 2.
``` C
#include "KerbalSimpit.h"
#define PIN_SWITCH 2
KerbalSimpit mySimpit(Serial);
bool oldSwitchState = HIGH;

void setup() 
{
	Serial.begin(115200);
	pinMode(PIN_SWITCH, INPUT_PULLUP);

	while (!mySimpit.init()) delay(100);
}

void loop() 
{
	if(digitalRead(PIN_SWITCH) == LOW) 
	{
		if(oldSwitchState == HIGH) 
		{
			Timewarp myWarpLevel;
			myWarpLevel = TIMEWARP_NEXT_MANEUVER;
			mySimpit.send(TIMEWARP_MESSAGE, myWarpLevel);
		}
		oldSwitchState = LOW;
	}
	else oldSwitchState = HIGH;
	delay(100); //Debounce switch
}
```

## Expected behaviour

The time warp rate should be set to something reasonable to warp to the desired point in time, when telling simpit to warp to a specific point in time (maneuver, SOI, Ap/Pe)

**Comment by PBechon** [comment]
I was able to reproduce it. The function we use to warp to a given time is line 221 of WarpControl.cs : `TimeWarp.fetch.WarpTo(UT, MAX_WARP_RATE, MIN_WARP_RATE)`. There is nothing that should prevent it to work here, but the log indicate that the max warp rate is 1 (probably why no warping occurs). No idea why ...

Replacing it with the default values for warp rate seems to solve the issue : `TimeWarp.fetch.WarpTo(UT)`.

Currently testing it before merging.

**Comment by bespezzano** [comment]
Have you tried using fixed rates just for testing?

**Comment by PBechon** [comment]
Those parameter are not behaving as described it seems. Since the default values seems to work, lets keep those.

Fixed in 39139b2e87fd8985ecdf97a423f1118e1eba3b69
